src/components/ui/icon-button/icon-button.test.js

```diff
@@ -58,4 +67,49 @@ describe('IconButton', () => {
 
     expect(wrapper.find('.pipeline-toolbar__label-right').length).toBe(1);
   });
+
+  it('show tooltip', () => {
+    jest.useFakeTimers();
+    const setIsTooltipVisible = jest.fn();
+    jest.useFakeTimers();
```

Comment:
why do you need to set `useFakeTimers()` twice?

Reply:
Yes, thanks for noticing this, it's added by mistake.